lib/galaxy/tools/__init__.py

```diff
@@ -191,6 +192,16 @@
     "winSplitter": packaging.version.parse("1.0.1"),
 }
 
+REQUIRE_FULL_DIRECTORY = {
```

Comment:
Handling this doesn't appear to be implemented?

Reply:
I think it is - see comment below?

Reply:
Have you tried it and it doesn’t work?

Reply:
Nevermind, I see how it works now, thanks.